File: java-apps/kq-common/src/main/java/com/kafkaquest/kq/common/util/models/events/responsedata/PieceRemovedEventResponseData.java
package com.kafkaquest.kq.common.util.models.events.responsedata;

import lombok.AllArgsConstructor;
import lombok.Data;
import lombok.EqualsAndHashCode;
import lombok.NoArgsConstructor;

import java.util.UUID;

@Data
@EqualsAndHashCode(callSuper = false)
@NoArgsConstructor
@AllArgsConstructor
public class PieceRemovedEventResponseData extends ResponseData {
    private UUID pieceId;
    private int removedRowPosition;
    private int removedColumnPosition;
}

File: kq-projection-dotnet/kq-projection/Common/Models/Events/PieceRemovedEventResponseData.cs
using System.Text.Json.Serialization;

namespace KQ_Projection.Common.Models.Events
{
    public class PieceRemovedEventResponseData : ResponseData
    {
        [JsonPropertyName("pieceId")]
        public Guid pieceId { get; set; }

        [JsonPropertyName("removedRowPosition")]
        public int removedRowPosition { get; set; }


        [JsonPropertyName("removedColumnPosition")]
        public int removedColumnPosition { get; set; }
    }
}
